packages/integration_test/ios/Classes/FLTIntegrationTestRunner.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+
+@class UIImage;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^FLTIntegrationTestResults)(SEL nativeTestSelector, BOOL success, NSString *_Nullable failureMessage);
```

Comment:
This is almost the same as https://github.com/flutter/flutter/pull/88013 except it's passing back a test selector `testVerifyTest` instead of the test name `verify test`. This allows us to do more of the test name casing logic in the plugin instead of the macro.